packages/mobile/src/harmony-native/components/layout/Paper/Paper.tsx

```diff
@@ -83,16 +83,25 @@ export const Paper = forwardRef<View, PaperProps>((props, ref) => {
 
   const flexProps = { backgroundColor, borderRadius, shadow }
 
+  const universalStyles: StyleProp<ViewStyle> = { overflow: 'hidden' }
```

Comment:
whoa so now all `Paper` components will have `overflow: hidden`? i'm down but maybe worth flagging to the team.

Reply:
yeah this brings it in line with web harmony Paper. Paper's meant to be a container so overflow is generally not desirable. I'll call out the change in the harmony channel.